spring-batch-infrastructure/src/main/java/org/springframework/batch/item/file/FlatFileItemWriter.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2006-2018 the original author or authors.
+ * Copyright 2006-2019 the original author or authors.
```

Comment:
This change should be rolled back since the file has not been changed.

Reply:
fixed